.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- [ ] Kindly make sure that`node_modules` is also presented in `.gitignore` file in order to be ignored by git.